code_challenges/adhoc/rotate_v2.py

```diff
@@ -0,0 +1,8 @@
+def rotate(matrix):
+    rotated = []
+    for i in range(len(matrix)):
+        row = []
+        for j in range(len(matrix) - 1, -1, -1):
+            row.append(matrix[j][i])
+        rotated.append(row)
+    return rotated
```

Comment:
close, add these functions in the same file as the other.  So instead of new files named rotate_v2, you will have a new function named rotate_v2 in the same file as the original.

Reply:
Also - have to remove the other files rotate_v2 and rotate_v3 since they are now functions in this file, and add the same tests that are for the other function for these functions too.